terraform/vars.tf

```diff
@@ -0,0 +1,90 @@
+variable "region" {
+  description = "The AWS region to create resources in."
+  default = "us-east-1"
+}
+
+variable "remote_state_backend_type" {
+  description = “The remote state backend, if you’re planning to hook this up to an existing DevSecOps-Infrastructure deployment.”
+  type = "string"
+  default = “s3”
+}
+
+variable "remote_state_backend_bucket" {
+  description = “The remote state backend bucket.”
+  type = "string"
+}
+
+variable "remote_state_backend_key" {
+  description = “The remote state key for the terraform.tfstate file. Ex: devsecops-infrastructure/terraform.tfstate”
+  type = "string"
+}
+
+variable "sg_vpc_id" {
+  description = “The VPC ID for the Jenkins security group. If you do not specify this, the deployment will attempt to use the vpc-mgmt VPC ID from the DevSecOps-Infrastructure deployment. You MUST have the remote_state_backend variables defined above for that to work.”
+  type = "string"
```

Comment:
Any reason not to go ahead and define the default as ""?

Reply:
Nope, can do :)